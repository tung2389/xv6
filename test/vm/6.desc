Test 0 or negative length