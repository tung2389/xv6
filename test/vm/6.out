XV6_TEST_OUTPUT invalid len
